addons/nuxeo-ai-core/index.js

```diff
@@ -115,13 +115,14 @@ const AISuggestionManager = (() => {
       }
       const aiModels = layout.document.contextParameters && layout.document.contextParameters.aiModels;
       const isModelInput = path && aiModels && aiModels.inputs.includes(path.replace('document.properties.', ''));
+      const noSpecificPropertyChanged = !path || (path && path.length <= 'document.properties'.length);
```

Comment:
It would be safer to actually check the path matches `document.properties` and not just its length:
```suggestion
      const noSpecificPropertyChanged = !path || 'document.properties'.startsWith(path);
```
nit: I'd even name name it `noPropertyChanged ` or `noDocPropertyChanged` since the "specific" doesn't add much.